ui/DesignSystem/Component/AdditionalActionsDropdown.svelte

```diff
@@ -1,30 +1,38 @@
 <script>
   import { fade } from "svelte/transition";
 
+  import {
```

Comment:
each dropdown shouldn't have import all this stuff - it'll exist in a wrapper for ergonomics